### PR TITLE
Refactoring lmnet/executor/export.py

### PR DESCRIPTION
**Pull request by ruimashita**
It is hard to understand some variable or function names like `npy_output_dir` and `_save_npy` etc in  `lmnet/executor/export.py`. I refactoring function and variable and comment names, more easily to understand for developer.

Related PR:
https://github.com/blue-oil/blueoil/pull/184#issuecomment-470429112

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by ruimashita** [comment]
@lm-jira @nlpng 
Reviwe please 👍

File: lmnet/executor/export.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def _pre_process(raw_image, pre_processor, data_format):
     return image
 
 
-def _save_npy(image_path, output_dir, image, raw_image, all_outputs, image_size):
+def _save_inference_values_foreach_operation_as_npy(image_path, output_dir, image, raw_image, all_outputs, image_size):
```
Comment (nlpng) [inline lmnet/executor/export.py]: I think the name may be too long, makes it hard to read.

Comment (ruimashita) [reply lmnet/executor/export.py]: @nlpng 
How is `_save_all_operation_outputs()` ?

Comment (nlpng) [reply lmnet/executor/export.py]: Thanks, this sounds more recognizable.

**Comment by iizukak** [comment]
@nlpng Review again 🙇

**Comment by iizukak** [comment]
@ruimashita This PR is still WIP? and Please solve conflict 🙇

**Comment by ruimashita** [comment]
@ruimashita 
Sorry late reply, this is still WIP

**Comment by ruimashita** [comment]
@iizukak Ready to review :pray:

**Pull request merged**